src/server/common.h

```diff
@@ -4,17 +4,21 @@
 
 #pragma once
 
-#include <boost/fiber/mutex.hpp>
 #include <absl/strings/ascii.h>
 #include <absl/strings/str_cat.h>
 #include <absl/types/span.h>
 
+#include <boost/fiber/mutex.hpp>
 #include <string_view>
 #include <vector>
 
 #include "facade/facade_types.h"
 #include "facade/op_status.h"
 
+extern "C" {
```

Comment:
I prefer to avoid leaking `redis/` includes, so please move `Matches(...)` implementation to cc file.

Reply:
Sure, I will move match to the cc file. Note that I will not be a constexpr function anymore